vue2-vuetify/src/layouts/ArrayLayoutRenderer.vue

```diff
@@ -289,10 +295,12 @@ const controlRenderer = defineComponent({
     const currentlyExpanded = ref<null | number>(
       control.appliedOptions.value.initCollapsed ? null : 0
     );
+    const focusable =
+      control.vuetifyProps('v-expansion-panels')?.focusable ?? true;
```

Comment:
Sadly we are in Vue2 in which the specific attribute always takes precedence over the `v-bind`. In Vue 3 we could just set the defaults as specific attributes and let the user overwrite them via `v-bind` by placing it later in the template as the attributes.

Instead of extracting this into own component state values I would handle them within some kind of props wrapper e.g.

```
const expansionPanelsProps = computed(() => merge({ focusable: true}, control.vuetifyProps('v-expansion-panels'));
```
and then bind this in `v-bind` and remove the `focusable` and `flat` attributes.

However if you prefer state values then that is also fine with me. We need to check whether they are correctly updated when the user changes the UI Schema or config on the fly.